src/App.jsx

```diff
@@ -13,6 +14,8 @@ function App() {
                 <Switch>
                     <Route path="/signup" component={Signup} />
                     <Route path="/signin" component={Signin} />
+                    <Route path="/feeds" component={Feed} />
+                    <Route path="/" component={Feed} />
                 </Switch>
             </Router>
```

Comment:
it is better to separate this to a component

Reply:
how do u mean boss

Reply:
create a new component called `Route` and import it here

Reply:
I think it  is fairly understandablee and still static